cove_ocds/management/commands/ocds_cli.py

```diff
@@ -13,15 +13,20 @@ class Command(CoveBaseCommand):
     help = 'Run Command Line version of Cove OCDS'
 
     def add_arguments(self, parser):
-        parser.add_argument('--schema-version', '-s', default='', help='Version of the schema to validate the data')
-        parser.add_argument('--convert', '-c', action='store_true', help='Convert data from nested (json) to flat format (spreadsheet) or vice versa')
+        parser.add_argument('--schema-version', '-s', default='',
+                            help='Version of the schema to validate the data')
+        parser.add_argument('--convert', '-c', action='store_true',
+                            help='Convert data from nested (json) to flat format (spreadsheet) or vice versa')
+        parser.add_argument('--cache', '-z', action='store_true',
+                            help='Cache the remote schema and extensions in memory')
```

Comment:
I would say by default the cli should cache. So I am not sure there is a need for that argument.
We could have an option --nocache but not sure its necessary.

Reply:
Will change to default caching.

> We could have an option --nocache but not sure its necessary.

Prob not necessary.